main.go

```diff
@@ -102,25 +111,33 @@ func init() {
 	flag.Var(&exclude, "exclude", `path to exclude (can be specified multiple times).`)
 	flag.BoolVar(&dryRun, "d", false, `skips rewriting files and returns exitcode 1 if any discrepancies are found.`)
 	flag.StringVar(&extension, "ext", defaultExt, "sets the file extension to scan for.")
+	flag.StringVar(&license, "license", defaultLicense, "sets the license type to check: ASL2, Elastic")
 	flag.Usage = usageFlag
 	flag.Parse()
 	args = flag.Args()
-	for i := range Header {
-		headerBytes = append(headerBytes, []byte(Header[i])...)
-		headerBytes = append(headerBytes, []byte("\n")...)
-	}
 }
 
 func main() {
-	err := run(args, exclude, extension, dryRun, os.Stdout)
+	err := run(args, license, exclude, extension, dryRun, os.Stdout)
 	if err != nil && err.Error() != "<nil>" {
 		fmt.Fprint(os.Stderr, err)
 	}
 
 	os.Exit(Code(err))
 }
 
-func run(args, exclude []string, ext string, dry bool, out io.Writer) error {
+func run(args []string, license string, exclude []string, ext string, dry bool, out io.Writer) error {
+	header, ok := Headers[license]
+	if !ok {
+		return &Error{err: fmt.Errorf("unkown license: %s", license), code: errUnknownLicense}
```

Comment:
s/unkown/unknown/

Reply:
nit: Maybe we could add the valid licenses? This will make the error message easily actionable.